spec/rmagick/image/channel_entropy_spec.rb

```diff
@@ -1,7 +1,7 @@
 RSpec.describe Magick::Image, '#channel_entropy' do
   let(:img) { Magick::Image.read(IMAGES_DIR + '/Button_0.gif').first }
 
-  it 'returns a channel entropy' do
+  it 'returns a channel entropy', skip: unsupported?('6.9.0') do
```

Comment:
This kind of reads like "skip if version '6.9.0' is unsupported". I'm trying to think how we could make it clearer that it is testing a method that is supported on or after `6.9.0`. One thought is something like:

```rb
it 'returns a channel entropy', supported_after('6.9.0') do
```

Then the method below would be something more like:

```rb
def supported_after(version)
  :skip if Gem::Version.new(magick_lib_version) < Gem::Version.new(version)
end
```

Another thought is to also test the other case. I was looking at how [`rspec-rails` handles different Rails versions in their tests](https://github.com/rspec/rspec-rails/blob/master/spec/generators/rspec/mailer/mailer_generator_spec.rb#L16-L21), and it looks like they put the entire test block in a conditional. If we were to mimic that, it might be something like:

```rb
if magick_lib_version.to_f > 6.9
  it 'returns a channel entropy' do
    ...
  end
else
  it 'raises a NotImplementedError' do
    ...
  end
end
```